### DISCUSSION TITLE
Combine two fonts

### DISCUSSION THREAD

**Discussion by thr33bricks** [olikraus/u8g2#1891] [Q&A] [answered]
Hello guys! I would like to combine **u8g2_font_10x20_t_cyrillic** and **u8g2_font_unifont_t_emoticons**. I tried just combining the font data but every time I tried to access a character from the second half of the data my board restarted. Obviously this method is not working. Any help would be appreciated!

**Comment by olikraus** [comment, marked as answer]
You need to combine the .bdf source for those fonts to a new .bdf file. Then recreate the u8g2 font with bdfconv.exe (see u8g2 FAQ)

**Reply by thr33bricks** [reply]
Thank you very much!
